apps/src/templates/courseOverview/VerifiedResourcesNotification.js

```diff
@@ -1,21 +1,28 @@
 import PropTypes from 'prop-types';
-import React from 'react';
+import React, {PureComponent} from 'react';
 import Notification, {NotificationType} from '@cdo/apps/templates/Notification';
 import i18n from '@cdo/locale';
 
-export default function VerifiedResourcesNotification({width}) {
-  return (
-    <Notification
-      type={NotificationType.information}
-      notice={i18n.verifiedResourcesNotice()}
-      details={i18n.verifiedResourcesDetails()}
-      buttonText={i18n.learnMore()}
-      buttonLink="https://support.code.org/hc/en-us/articles/115001550131"
-      dismissible={true}
-      width={width}
-    />
-  );
+export default class VerifiedResourcesNotification extends PureComponent {
```

Comment:
For my own knowledge: why did you change this from a function to a PureComponent?

Reply:
It just seemed more in line with how it was being used especially when I added tests for it.